docs/calico-with-docker/docker-network-plugin/CalicoAsService.md

```diff
@@ -0,0 +1,72 @@
+<!--- master only -->
+> ![warning](../../images/warning.png) This document applies to the HEAD of the calico-containers source tree.
+>
+> View the calico-containers documentation for the latest release [here](https://github.com/projectcalico/calico-containers/blob/v0.14.0/README.md).
+<!--- else
+> You are viewing the calico-containers documentation for release **release**.
+<!--- end of master only -->
+
+# Calico as a Docker network plugin service
+
+When starting Calico as a Docker network plugin, the `calicoctl node --libnetwork`
+command will start up both the `calico/node` image (core Calico) and the
+`calico/node-libnetwork` image for Docker network integration.
+
+In some cases, you may want to run the `calico/node-libnetwork` Docker image as a separate service.  For example, if you choose to run `calicoctl node --detach=false --libnetwork`, Calico will start and attach to the `calico/node` container
+without starting the `calico/node-libnetwork` container.
```

Comment:
Are you sure this is what it does?  I thought it rejected the request - basically you can't use the detach option with libnetwork.